### ISSUE TITLE
Product number field in checkout uses placeholder as label.

### ISSUE THREAD

**Issue by tobiasberge** [shopware/platform#14834]
### Technical TODO

The "Product number" field in the checkout does not have a label and uses the placeholder as label instead. There is a `visually-hidden` label in place. However, it is also inconsistent to the "Promo code" field which has a visible label.

The "Product number" field should also have a visible label similar to "Promo code".

<img width="1381" height="571" alt="Image" src="https://github.com/user-attachments/assets/4e7a7817-bb64-469e-9aa5-d7bf09887eaa" />

**Comment by En0Ma1259** [comment]
- Related https://github.com/shopware/shopware/issues/14796 (maybe one fix)
